charts/reporting-operator/templates/custom-resources/report-queries/pvc-capacity.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportGenerationQuery
+metadata:
+  name: "pvc-capacity-raw"
+    labels:
+      operator-metering: "true"
+    {{- block "extraMetadata" . }}
+    {{- end }}
+spec:
+  reportDataSources:
+  - "pvc-capacity-bytes"
+  columns:
+  - name: namespace
+    type: string
+  - name: persistentvolumeclaim
+    type: string
+  - name: pvc_capacity_bytes
+    type: double
+    unit: bytes
+  - name: timeprecision
+    type: double
+    unit: seconds
+  - name: pvc_capacity_byte_seconds
+    type: double
+    unit: byte_seconds
+  - name: timestamp
+    type: timestamp
+    unit: date
+  query: |
+    SELECT
+        labels['exported_namespace'] as namespace,
+        labels['persistentvolumeclaim'] as persistantvolumeclaim,
+        amount as pvc_capacity_bytes,
+        timeprecision,
+        amount * timeprecision as pvc_used_byte_seconds,
+        "timestamp",
```

Comment:
Don't forget to select `dt` and add it to your columns as a string. This is the partition for the data, and other queries querying the view for this raw query can filter on `dt` to improve performance. The partition is just a string in the form of `2018-11-28` which is the day for the metric.